src/pytest_databases/docker/minio.py

```diff
@@ -0,0 +1,109 @@
+import os
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+from urllib.error import URLError
+from urllib.request import Request, urlopen
+
+import pytest
+from minio.api import Minio
+
+from pytest_databases.docker import TRUE_VALUES
+from pytest_databases.helpers import get_xdist_worker_num
+from pytest_databases.types import ServiceContainer, XdistIsolationLevel
+
+if TYPE_CHECKING:
+    from collections.abc import Generator
+
+    from pytest_databases._service import DockerService
+
+
+DEFAULT_MINIO_ACCESS_KEY = os.getenv("MINIO_ACCESS_KEY", "minio")
+DEFAULT_MINIO_SECRET_KEY = os.getenv("MINIO_SECRET_KEY", "minio123")
+DEFAULT_MINIO_SECURE = os.getenv("MINIO_SECURE", "false").lower() in TRUE_VALUES
```

Comment:
I think we should stay consistent and do this configuration via fixtures

Reply:
Updated